daemon/logger/factory.go

```diff
@@ -119,6 +119,7 @@ func GetLogDriver(name string) (Creator, error) {
 var builtInLogOpts = map[string]bool{
 	"mode":            true,
 	"max-buffer-size": true,
+	logInfoKey:        true,
```

Comment:
This should be added to the API history (mention of the new option); https://github.com/moby/moby/blob/master/docs/api/version-history.md